script/down

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
 set -euo pipefail
-CLUSTER="example"
-doctl k8s cluster delete ${CLUSTER}
\ No newline at end of file
+CLUSTER="kb-example"
+# doctl k8s cluster delete ${CLUSTER}
+LBID=`doctl compute load-balancer list | grep -wo '^[0123456789abcdef\-]*'`
```

Comment:
This might be a bit dangerous as it may delete several LBs, and not only the one associated with the example cluster.

I think a better way would be to delete the `doks-example` Service using `kubectl` (i.e., `kubectl delete service doks-example`) and wait 5 seconds for the delete request to have reached the LB API  before deleting the cluster.